linkerd/app/src/env.rs

```diff
@@ -96,8 +96,13 @@ pub const ENV_METRICS_RETAIN_IDLE: &str = "LINKERD2_PROXY_METRICS_RETAIN_IDLE";
 
 const ENV_INGRESS_MODE: &str = "LINKERD2_PROXY_INGRESS_MODE";
 
-const ENV_INBOUND_DISPATCH_TIMEOUT: &str = "LINKERD2_PROXY_INBOUND_DISPATCH_TIMEOUT";
-const ENV_OUTBOUND_DISPATCH_TIMEOUT: &str = "LINKERD2_PROXY_OUTBOUND_DISPATCH_TIMEOUT";
+const ENV_INBOUND_HTTP_BUFFER_CAPACITY: &str = "LINKERD2_PROXY_INBOUND_HTTP_BUFFER_CAPACITY";
+const ENV_INBOUND_HTTP_FAILFAST_TIMEOUT: &str = "LINKERD2_PROXY_INBOUND_HTTP_FAILFAST_TIMEOUT";
+
+const ENV_OUTBOUND_TCP_BUFFER_CAPACITY: &str = "LINKERD2_PROXY_OUTBOUND_TCP_BUFFER_CAPACITY";
```

Comment:
nit/tioli: not sure about calling this "TCP_BUFFER_CAPACITY" --- it seems like it could be misinterpreted to mean the capacity of the TCP send/recv buffers? maybe we should be calling these `QUEUE_CAPACITY` instead? that would be using a different name than `tower::buffer`, but it seems less easy to mix up with other "TCP buffer" configurations...

Reply:
or even "TCP_CONNECT_BUFFER" or something, maybe?

Reply:
i like queue.